src/main/java/calculatorFunction/GetInput.java

```diff
@@ -0,0 +1,56 @@
+package calculatorFunction;
+
+import java.util.Scanner;
+
+public class GetInput {
+
+    public static String inputString() {
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
+        return str;
+    }
+
+    public static String[] parsing(String str) {
+        // 1 + 3 - 5
+        String[] array = new String[100];
+        int it = 0;
+        int strIt = 0;
+        array[it] = getInteger(str, strIt);
+        strIt = array[it].length() + strIt;
+        it++;
+
+        while (strIt != str.length()) {
+            array[it++] = getString(str, strIt);
+            strIt++;
+            array[it] = getInteger(str, strIt);
+            strIt = array[it].length() + strIt;
+            it++;
+        }
+
+        return array;
+    }
+
+    public static String getInteger(String str, int idx) {
+        String string = "";
+        for (int i = idx; i < str.length(); i++) {
+            if (str.charAt(i) >= '0' && str.charAt(i) <= '9') {
+                string += str.charAt(i);
+            } else {
+                break;
+            }
+        }
+
+        if (string.length() == 0) {
+            return "ERROR";
+        } else if (string.charAt(0) == '0' && string.length() > 1) {
+            return "ERROR";
+        }
+
+        return string;
+    }
+
+    public static String getString(String str, int idx) {
+        return Character.toString(str.charAt(idx));
```

Comment:
```suggestion
        return String.valueOf(str.charAt(idx));
```
위와 같이 변경하시는게 더 좋을 것 같아요 🤗
이렇게 변경한다면 `getString` 메서드를 따로 분리할 필요 없이 바로 필요한 위치에서 String.valueOf(str.charAt(idx))를 호출하는 것이 좋을 것 같아요 🤔